test-data/exceptions.test

```diff
@@ -187,3 +187,82 @@ L12:
 L13:
     dec_ref r10
     goto L7
+
+[case testTryFinally2Asdf]
+def a() -> str:
+    try:
+        print()
+        return 'hi'
+    finally:
+        pass
```

Comment:
I'd prefer to have a non-empty body here. Maybe put something that can fail here?